crates/context/config/src/client/config.rs

```diff
@@ -71,5 +72,13 @@ pub enum Credentials {
     Near(NearCredentials),
     Starknet(StarknetCredentials),
     Icp(IcpCredentials),
-    Stellar(StellarCredentials),
+    Raw(RawCredentials),
```

Comment:
Stellar and Evm imported but not used anywhere? How does this update work? If they are same creds object to we need these two imports then?